rpi/page.py

```diff
@@ -111,39 +111,39 @@ class RegisterPage(Page):
+		# Create a register frame
+		register_frame = tk.Frame(self, height=660, width=1320, bg="#fff", bd=2, relief="solid")
+		register_frame.pack_propagate(0)
 		
-		self.enrollment_result_string = tk.StringVar(value="")
-		self.enrollment_result_message = tk.Label(self, textvariable=self.enrollment_result_string, wraplength=500, justify="center", bg="#fff", font="helvetica 14")
+		self.label = tk.Label(register_frame, text="Register", font="helvetica 20 bold", bg="#fff")
+		self.label.pack(pady=50)
 
-		self.fingerprint_enrollment_btn = tk.Button(self, text="Fingerprint Enrollment", height=1, width=16, command=lambda: enroll_fingerprint(self))
+		self.username_label = tk.Label(register_frame, bg="#fff", text="Username")
+		self.username_input = tk.Entry(register_frame, width=20, bg="#fff", bd=1, relief="solid")
+		self.password_label = tk.Label(register_frame, bg="#fff", text="Password")
+		self.password_input = tk.Entry(register_frame, show="*", width=20, bg="#fff", bd=1, relief="solid")
+		
+		self.register_user_btn = tk.Button(register_frame, text="Register", height=1, width=8, command=lambda: attempt_register(self, self.username_input.get(), self.password_input.get(), self.fingerprintId))
+		self.fingerprint_enrollment_btn = tk.Button(register_frame, text="Fingerprint Enrollment", height=1, width=16, command=lambda: enroll_fingerprint(self))
 		
-		self.register_success_label = tk.Label(self, text="Registered successfully!", font="helvetica 15", bg="#fff", foreground="green")
+		self.register_success_label = tk.Label(register_frame, text="Registered successfully!", font="helvetica 15", bg="#fff", foreground="green")
 
-		self.first_img_instruction_label = tk.Label(self, text="Place your finger on the scanner...", font="helvetica 15", bg="#fff")
-		self.second_img_instruction_label = tk.Label(self, text="Place the same finger again...", font="helvetica 15", bg="#fff")
-		self.img_error_label = tk.Label(self, text="Error while capturing finger image", font="helvetica 15", bg="#fff", foreground="red")
-		self.storage_error_label = tk.Label(self, text="Error while storing the fingerprint", font="helvetica 15", bg="#fff", foreground="red")
-		self.enrollment_success_label = tk.Label(self, text="Enrolled successfully!", font="helvetica 15", bg="#fff", foreground="green")
+		self.first_img_instruction_label = tk.Label(register_frame, text="Place your finger on the scanner...", font="helvetica 15", bg="#fff")
+		self.second_img_instruction_label = tk.Label(register_frame, text="Place the same finger again...", font="helvetica 15", bg="#fff")
+		self.img_error_label = tk.Label(register_frame, text="Error while capturing finger image!", font="helvetica 15", bg="#fff", foreground="red")
+		self.storage_error_label = tk.Label(register_frame, text="Error while storing the fingerprint!", font="helvetica 15", bg="#fff", foreground="red")
```

Comment:
Not the first time I see this. Exclamation point on every messages is really really annoying to users. I'd say put a dot there and encourage them to try again/contact admin is better here

Reply:
Oh okay, should be updated now